src/index.ts

```diff
@@ -64,6 +64,11 @@ export interface IClientConfig {
    * backoff. (Default: 7)
    */
   numPollRequestRetries?: number;
+
+  /**
+   * Skip the first poll for new configurations during initialization. (default: false)
+   */
+  skipInitialPoll?: boolean;
```

Comment:
🔥

Reply:
Just a thought: "poll" implies a request made repeatedly at intervals. `skipInitialRequest`?

Reply:
Good idea - I was being too pedantic with the mechanics of the internal poller. For customers they are not going to know that and the naming you suggested is more clear 👍 👍